### DISCUSSION TITLE
Cake Addin for Asciidoctor

### DISCUSSION THREAD

**Discussion by ap0llo** [cake-build/cake#4196] [Extension requests]
Hi,

I have been working on an addin that provides aliases for [Asciidoctor](https://docs.asciidoctor.org/asciidoctor/latest/) and [Asciidoctor PDF](https://docs.asciidoctor.org/pdf-converter/latest/).

Would it be possible to create a "Cake.Asciidoctor" repo in the cake-contrib organization for that addin?

**Comment by nils-a** [comment]
Hi @ap0llo, here are the instructions for the transfer:

* For GitHub, this is a multi-step operation due to how GitHub manages permissions for organizations.

  * Verify that I'm a member of the core Cake maintainer team on the official website: https://cakebuild.net/docs/team/
 
  * Transfer the repo to my GitHub user nils-a ([instructions](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository))
  * I transfer it to the [Cake Contrib](https://github.com/cake-contrib) organization
  * ~~You accept the invite to join the organization (you should have received an email)~~ You're already in the Cake Contrib organization
  * I add you as admin in your repo, within the org

**Reply by ap0llo** [reply]
Sorry for the late response, but since the addin is already published and I don't expect much changes to it in the foreseeable future, I've decided to keep the repo under my GitHub account for now.
